app/controllers/predicates/AuthorisedAction.scala

```diff
@@ -104,15 +112,15 @@ class AuthorisedAction @Inject()(
               block(User(mtditid, Some(arn),nino))
             case None =>
               logger.debug("[AuthorisedAction][CheckAuthorisation] Agent with no HMRC-AS-AGENT enrolment. Rendering unauthorised view.")
-              Future.successful(Forbidden("")) //TODO add agent unauthorised page
+              Future.successful(Forbidden(agentAuthErrorPage("Agent Unauthorized", "You're not enrolled for the agent service")))
```

Comment:
Swap to passing in flag / boolean to decide which content to render, move content to view.